examples/webgl_interactive_instances_gpu.html

```diff
@@ -0,0 +1,438 @@
+
+						scene.add( object );
+
+						//
+
+						pickingData[ i + 1 ] = object;
+
+						var pickingObject = object.clone();
+
+						if( useSingleMaterial ){
+
+							object.material.hasDynamicUniforms = true;
+							object.userData[ "color" ] = color;
+
+							pickingObject.material = pickingMaterial;
+							pickingObject.material.hasDynamicUniforms = true;
+							pickingObject.userData[ "pickingColor" ] = i + 1;
+
+						}else{
+
+							object.material = material.clone();
+							object.material.color.setHex( color );
+							object.material.hasDynamicUniforms = false;
+							materialList.push( object.material );
+
+							pickingObject.material = pickingMaterial.clone();
+							pickingObject.material.hasDynamicUniforms = false;
+							pickingObject.material.uniforms.pickingColor.value.setHex( i + 1 );
+							materialList.push( pickingObject.material );
+
+						}
+
+						pickingScene.add( pickingObject );
+
+					}
+
+					//
+
+					render();
+
+					console.timeEnd( "init mesh" );
+
+					console.log( "material count", materialList.length );
+					console.log( renderer.info.memory )
+					console.log( renderer.info.render )
+
+				} );
+
+			}
+
+			function init() {
+
+				container = document.getElementById( "container" );
+
+				var pointLight = new THREE.PointLight( 0xffffff, 1, 0 );
+				pointLight.position.z = 1000;
+
+				camera = new THREE.PerspectiveCamera(
+					70, window.innerWidth / window.innerHeight, 1, 10000
+				);
+				camera.position.z = 1000;
+				camera.add( pointLight );
+
+				//
+
+				pickingTexture = new THREE.WebGLRenderTarget(
+					window.innerWidth, window.innerHeight
+				);
+				pickingTexture.generateMipmaps = false;
+				pickingTexture.minFilter = THREE.NearestFilter;
+
+				//
+
+				var boxGeo = new THREE.BoxGeometry( 1, 1, 1 );
+
+				highlightBox = new THREE.Group();
+
+				highlightBox.add(
+					new THREE.Mesh(
+						boxGeo,
+						new THREE.MeshLambertMaterial( {
+							emissive: 0xffff00,
+							transparent: true,
+							opacity: 0.5,
+							side: THREE.BackSide
+						} )
+					)
+				);
+
+				highlightBox.add(
+					new THREE.Mesh(
+						boxGeo,
+						new THREE.MeshLambertMaterial( {
+							emissive: 0xffff00,
+							transparent: true,
+							opacity: 0.5,
+							side: THREE.FrontSide
+						} )
+					)
+				);
+
+				//
+
+				renderer = new THREE.WebGLRenderer( {
+					antialias: true,
+					alpha: true
+				} );
+				renderer.setClearColor( 0xffffff );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.sortObjects = false;
+				container.appendChild( renderer.domElement );
+
+				//
+
+				controls = new THREE.TrackballControls(
+					camera, renderer.domElement
+				);
+				controls.rotateSpeed = 1.0;
+				controls.zoomSpeed = 1.2;
+				controls.panSpeed = 0.8;
+				controls.noZoom = false;
+				controls.noPan = false;
+				controls.staticMoving = true;
```

Comment:
Do you mean `controls.staticMoving = false;`, since you have specified a damping factor? If you don't care about damping, then I expect you can remove your animation loop.